shell/machine-config/harvester.vue

```diff
@@ -169,6 +165,14 @@ export default {
       if (isEmpty(this.value.diskSize)) {
         this.value.diskSize = '40';
       }
+
+      try {
+        const { data: nodes } = await this.$store.dispatch('cluster/request', { url: `${ url }/${ NODE }s` });
+
+        this.allNodeObjects = nodes;
+      } catch (err) {
+        this.allNodeObjects = [];
+      }
```

Comment:
I think this part should also be put inside` if (clusterId && isImportCluster)`，  The external cluster does not need to request the node api.

Reply:
other LGTM.